specs/engage-digital_openapi3.yaml

```
@@ -6994,6 +6996,82 @@ paths:
       summary: Updating a user
       tags:
         - Users
+  /users/{userId}/sources_permissions:
+    get:
+      description: >-
+        This method renders the permissions the user has on each source.
+
+        Authorization​: only users that can view users. If the user affiliated
+        to the token has the manage_users_of_my_teams permission, only the signatures
+        of the users belonging to at least one of the teams he’s the leader of will be available.
+      operationId: getUserSourcesPermissions
+      parameters:
+        - in: path
+          name: userId
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserSourcesPermissions'
+          description: Success
+      summary: Getting a user's permissions on sources
+      tags:
+        - Users Sources Permissions
+    put:
+      description: >-
+        This method updates the permissions the user has on each source.
+
+        Authorization​: only users that can view users. If the user affiliated
+        to the token has the manage_users_of_my_teams permission, only the signatures
+        of the users belonging to at least one of the teams he’s the leader of will be available.
+      operationId: updateUserSourcesPermissions
+      parameters:
+        - in: path
+          name: userId
+          required: true
+          schema:
+            type: string
+        - description: >-
+            The parameter should be the id of a source, the value should be a string containing
+            the list of permissions you want to give to the user on a specific source (seprated by commas).
+
+            List of available permissions:
+              - read
+              - reply
+              - reply_with_html
+              - initiate_discussion
+              - destroy
+              - approval_required (if approval threshold is not enabled)
+              - approval_threshold_0
+              - approval_threshold_10
+              - approval_threshold_20
+              - approval_threshold_30
+              - approval_threshold_40
+              - approval_threshold_50
+              - approval_threshold_60
+              - approval_threshold_70
+              - approval_threshold_80
+              - approval_threshold_90
+              - approval_threshold_100
+          in: query
+          name: source_x_id

```

```suggestion
          name: "{source_x_id}"

```
We should make sure that people understand that this name is a placeholder and should be a content source id.

We should also add an example key for this param.

---

done in [dd29655](https://github.com/ringcentral/engage-digital-api-docs/pull/248/commits/dd296559e0aad011ab255346b51317e44477b2ab)